Update using-admin-roles.md (#8194)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/using-admin-roles.md

<!-- <sup>3</sup> Azure Active Directory admin center <<note that these are going to come later because they're related to Microsoft 365 Group management>> 
<sup>4</sup> Microsoft 365 Admin Center <<note that these are going to come later because they're related to Microsoft 365 Group management>> 
-->

>   |!NOTE| Set-csuser CMDLET will be loaded only if the user is a teams administrator, so for managing the enterprisevoice enabled it will be needed to give this role to the user 

Reviewer: ```suggestion
> [!NOTE]
> To load the **Set-CsUser** cmdlet, the user must be a Teams administrator, so to manage Enterprise Voice, this role must be given to the user.
```

<!-- <sup>3</sup> Azure Active Directory admin center <<note that these are going to come later because they're related to Microsoft 365 Group management>> 
<sup>4</sup> Microsoft 365 Admin Center <<note that these are going to come later because they're related to Microsoft 365 Group management>> 
-->
> [!NOTE]

> > To load the **Set-CsUser** cmdlet, the user must be a Teams administrator, so to manage Enterprise Voice, this role must be assigned to the user.

Reviewer: ```suggestion
> To load the **Set-CsUser** cmdlet, the user must be a Teams administrator, so to manage Enterprise Voice, this role must be assigned to the user.

```
